pkgs/development/tools/literate-programming/noweb/default.nix

```diff
@@ -54,7 +57,7 @@ let noweb = stdenv.mkDerivation rec {
         # NOTE: substituteInPlace breaks Icon binaries, so make sure the script
         #       uses (n)awk before calling.
         if grep -q nawk "$f"; then
-            substituteInPlace "$f" --replace "nawk" "${gawk}/bin/awk"
+            substituteInPlace "$f" --replace "nawk" "${nawk}/bin/awk"
```

Comment:
Will you please add `$out/lib/noweb/pipedocs` to this of files to patch?

Reply:
Ok. Was it added between the two versions?

Reply:
Thanks!